metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -1327,7 +1335,8 @@ def _build_input_measure_spec(
             offset_window=child_metric_offset_window,
             offset_to_grain=child_metric_offset_to_grain,
             cumulative_description=cumulative_description,
-            filter_specs=filter_specs,
+            after_aggregation_filter_specs=filter_spec_set.after_aggregation_filter_specs if filter_spec_set else (),
```

Comment:
Can we update this method to just take in the `filter_spec_set` instead of these two separate params?